airbyte-webapp/src/packages/cloud/views/layout/CloudMainView/CloudMainView.tsx

```diff
@@ -38,6 +39,7 @@ const CloudMainView: React.FC<React.PropsWithChildren<unknown>> = (props) => {
   const workspace = useCurrentWorkspace();
   const cloudWorkspace = useGetCloudWorkspace(workspace.workspaceId);
   const isAllowUpdateConnectorsEnabled = useFeature(FeatureItem.AllowUpdateConnectors);
+  const isShowAdminWarningEnabled = useFeature(FeatureItem.ShowCustomerWorkspaceWarning);
```

Comment:
Given we don't set it as a default value, I assume we plan to only enable that for airbyte employees via LD?

Reply:
Exactly, I think we should do opt-in first, then potentially add it as a default cloud feature if it goes over well